### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Angelesh1995**
This project was about to clone an old Apple website using backgrounds and gradients.

### CODE REVIEW

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @Angelesh1995      
#### Great Job so far, but there are a few more things required. 
### General
- [ ] The project should have a descriptive README.md
### Make changes and submit another code review request.
*Happy coding*

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
-This project consists of cloning an Apple website.
+# Background and Gradients
+
+This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.
+
+# Built With
+- HTML
+- CSS
+
+Link to live page
```
Comment (euqueme) [inline README.md]: Please provide a link  to the Live Demo or remove this line (#9)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
+.container {
+  width: 1440px;
```
Comment (euqueme) [inline style.css]: Delete line #2 as the container should have 100% of the screen width

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request Under The Review By Maria Eugenia Queme P. (TSE)

### Hello @Angelesh1995  :balloon: 

### Thank you for adding the Readme file as requested by the previous TSE :medal_sports: the gradient in your navbar is brilliant and  your use of backgrounds is stellar :star2:  still there are a couple of things to fix :hammer: 

## Body

- [ ] Add a [reset](https://meyerweb.com/eric/tools/css/reset/) at the beginning of your CSS file so your page doesn't set default browser properties as right now is adding 8px for the body

## Heroe Background

- [ ] It should take 100% of the page I put a suggestion in your code on how you could fix it and also avoids the scroll bar appearing in my screen :arrow_down: (something shifted after doing this? please fix it too) 

image

## Navbar

- [ ] The first button in your navbar is noticeably bigger than the other buttons, let's fix it

image

## Footer

the footer's right column (Site Map | Hot News | RSS Feeds | ...) should have more space in between the left column with the two rows like the original

- Yours

image

- Original

image

## Readme,

[OPTIONAL] Follow these [steps](https://help.github.com/en/github/working-with-github-pages/creating-a-github-pages-site#creating-your-site) to generate a Live Demo of your project and add the link to your Readme file

### That's it! :raised_hands: 

### Please :pray: make the changes :memo:  and submit back for a code review when you are done :rocket: 

Any questions? reach to me in slack ** @ Maru **

Happy Coding! :woman_technologist: 

PROJECT STATUS : REQUESTED CHANGES :vertical_traffic_light:

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Maria!  
### Your project looks :+1: amazing!
Just one thing is left:
* [ ] All linter errors should be fixed. Project can't be approved until then. 
### Please kindly fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by addod19 — APPROVED** [review]
## Status: Approved :+1:

Hi @Angelesh1995, **Congratulations on a job well done**. You have fix stickler errors as requested by the previous reviewer 😄 . You can now merge to master and submit project completion form.

Good luck with your next project 👍 
Happy coding 💻 
Daniel Larbi Addo

**Pull request merged**